modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/java/ScopeModelLoaderDelegate.java

```diff
@@ -96,9 +97,17 @@ void declareScope(ExtensionDeclarer extensionDeclarer,
                            Chain.class.getSimpleName(),
                            processorChain.stream().map(ExtensionParameter::getName).collect(toList())));
 
+
+    ParameterDeclarationContext declarationContext = new ParameterDeclarationContext(SCOPE, scope.getDeclaration());
     loader.getMethodParametersLoader().declare(scope,
                                                scopeMethod.getParameters(),
-                                               new ParameterDeclarationContext(SCOPE, scope.getDeclaration()));
+                                               declarationContext);
+
+    final List<ExtensionParameter> fieldParameters = scopeMethod.getEnclosingType().getParameters();
```

Comment:
this is pretty much the same code as for operations. CAn this be reused somehow?